draft-ietf-tls-tls13.md

```diff
@@ -2159,7 +2176,15 @@ Although X9.62 supports multiple point formats, any given curve
 MUST specify only a single point format. All curves currently
 specified in this document MUST only be used with the uncompressed
 point format (the format for all ECDH functions is considered
-uncompressed).
+uncompressed). Peers MUST validate each other's public value Y by ensuring
+that the point is a valid point on the elliptic
+curve. For all elliptic curves except for x25519 and x448, the appropriate
```

Comment:
@matthewdgreen the X25519/X448 RFC linked above does not mention Triple Handshake attacks.